### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by coforma-jamie**
As part of OCTO's initiative to improve accessibility documentation across VA.gov, this page has been restructured to  a) give a more thorough explanation of our approach, b) provide resources to non-VA teams, who use the VADS as a North Star, and c) give VA teams a more comprehensive starting point.

We're also working on a more internal-facing page that will be posted to Platform. We'll link to it here once that's complete.

If you have any questions, please reach out to Martha Wilkes. :) Thank you!

### CODE REVIEW

File: src/_about/accessibility/index.md [hunk 1]
```diff
@@ -2,12 +2,57 @@
 layout: category
 title: Accessibility
 permalink: /about/accessibility/
-intro-text: Current accessibility standards for the VA Design System
+intro-text: How to follow accessibility standards when using or contributing to the VA Design System (VADS)
 ---
 
-[Accessibility is core to all design decisions]({{ site.baseurl }}/about/principles#usable-by-everyone) made in our Design System. New components or patterns must complete the following to be admitted into the design system.
+VA is here to serve Veterans of the United States military, and [approximately 40% have an identified disability](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/accessibility/disability-statistics-veterans.md).
```
Comment (humancompanion-usds) [inline src/_about/accessibility/index.md:8]: We've tried to stop deep linking into GitHub. Can we find the original statistic and cite that?

File: src/_about/accessibility/index.md [hunk 2]
```diff
@@ -2,12 +2,57 @@
...
 
-[Accessibility is core to all design decisions]({{ site.baseurl }}/about/principles#usable-by-everyone) made in our Design System. New components or patterns must complete the following to be admitted into the design system.
+VA is here to serve Veterans of the United States military, and [approximately 40% have an identified disability](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/accessibility/disability-statistics-veterans.md). 
 
-## Foundational accessibility tests
+Therefore, [accessibility is core to all design decisions]({{ site.baseurl }}/about/principles#usable-by-everyone) made in our Design System. All components must be tested before admission into the Design System, and all VA products must be tested before launch.
+
+## Our holistic approach to accessibility
+
+The VA Design System provides accessible components, the guidance to implement these components, and the tools to customize and extend the design system accessibly.
```
Comment (humancompanion-usds) [inline src/_about/accessibility/index.md:14]: There should be some mention that we are built on the USWDS with a link to their Accessibility guidance: https://designsystem.digital.gov/documentation/accessibility/

File: src/_about/accessibility/index.md [hunk 3]
```diff
@@ -16,19 +61,31 @@ As part of staging reviews, new components must complete [foundational accessibi
...
+## Resources for Veteran-facing service teams
+
+### Learn how to make accessible products on the modernized VA.gov platform
+
+Visit the VA Platform site to learn about:
+
+- [Accessibility for developers](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-testing)
+- [Accessibility for researchers](https://depo-platform-documentation.scrollhelp.site/research-design/research-assistive-technology-sessions)
+- [How to prepare for an accessibility staging review](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review)
+- [Our plain language guidelines](https://design.va.gov/content-style-guide/plain-language/)
```
Comment (humancompanion-usds) [inline src/_about/accessibility/index.md:84]: Could you separate this one out since it is not on the VA Platform site, but instead in the Design System? Thanks.

File: src/_about/accessibility/index.md [hunk 4]
```diff
@@ -16,19 +61,31 @@ As part of staging reviews, new components must complete [foundational accessibi
...
+
 
-## Get accessibility guidance early from the CAIA team
+### Get accessibility guidance early from the CAIA team
```
Comment (humancompanion-usds) [inline src/_about/accessibility/index.md:87]: I thought Martha wanted to pull this into the VA Platform site and not mention CAIA here and that this section was what touched off an effort to figure out what goes where. If this information is available on VA Platform, I'd be okay with removing this and just linking to there. If not, perhaps it should move there and be the first link in the Resources section? If it does stay, it should probably go above the Resources section as it is required for VFS teams.

File: src/_about/accessibility/index.md [hunk 5]
```diff
@@ -16,19 +61,31 @@ As part of staging reviews, new components must complete [foundational accessibi
...
 4.   Keyboard navigation
 
-## Advanced testing with mobile and desktop screen readers
+### Advanced testing with mobile and desktop screen readers
 
-As part of staging reviews, new components must also complete [advanced testing with an accessibility specialist](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review#Prepareforanaccessibilitystagingreview-Advancedaccessibilitytests(recommended)advanced-testing) using a screen reader on both mobile and desktop. As a default, based on recommendations from the [Blinded Veterans Association](https://bva.org/), we recommend testing with:
+As part of staging reviews, new components must also complete [advanced testing with an accessibility specialist](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review#Prepareforanaccessibilitystagingreview-Advancedaccessibilitytests(recommended)advanced-testing) using a screen reader on both mobile and desktop. As a default, based on recommendations from the [Blinded Veterans Association](https://bva.org/), we recommend testing with the following at a minimum:
```
Comment (humancompanion-usds) [inline src/_about/accessibility/index.md:66]: I agree that we want teams to do this testing "at a minimum" but we don't really indicate here what a maximum effort would look like. Perhaps adding a list of other devices they could or should test with would be useful which would follow the minimum list? Or is the Note sufficient?

**Review by humancompanion-usds — CHANGES_REQUESTED** [review]
Overall this is great and I'm happy to see it and merge it very soon. Just a few questions for you to think through. Thanks!

File: src/_about/accessibility/index.md (hunk 1 above)
Comment (coforma-jamie) [reply src/_about/accessibility/index.md:8]: Sure! The stat comes from Statista, [Number of veterans in the United States in 2022, by service-connected disability status](https://www.statista.com/statistics/250316/us-veterans-by-disability-status/). If you combine all the rated Vets, and divide by the non-rated (or no-rating-reported) Vets, you get 39.5%.

How should we proceed - link to Statista directly?

File: src/_about/accessibility/index.md (hunk 2 above)
Comment (coforma-jamie) [reply src/_about/accessibility/index.md:14]: How about a paragraph after this:

The VA Design system was built on top of a fork of the U.S. Web Design System (USWDS), which prioritizes accessibility throughout. Learn more about [how USWDS practices accessibility](https://designsystem.digital.gov/documentation/accessibility/).

File: src/_about/accessibility/index.md (hunk 3 above)
Comment (coforma-jamie) [reply src/_about/accessibility/index.md:84]: By "separate out," do you mean not include in the list? I'd love to keep it here if possible, since this is a list of VA resources for different disciplines, regardless of where the content "lives".

Also for context, we're close to adding a VFS-specific accessibility landing page on Platform. At that time, the plan is to remove this list and replace it with a reference to the new landing page.

File: src/_about/accessibility/index.md (hunk 4 above)
Comment (coforma-jamie) [reply src/_about/accessibility/index.md:87]: The idea was to consider what audiences use the VADS, and I came up with two broad ones - VFS teams, and others who use this as a North Start but aren't a part of VA. Martha seemed fine to leave the reference here, as long as it was clear that it's for the VFS team audience, and not the global audience.

But I agree, putting it before the Resources makes a lot of sense! Is that something you do on your end, or can I do it from within this PR?

File: src/_about/accessibility/index.md (hunk 5 above)
Comment (coforma-jamie) [reply src/_about/accessibility/index.md:66]: Checking with Martha, will tell you her answer :)

File: src/_about/accessibility/index.md (hunk 1 above)
Comment (humancompanion-usds) [reply src/_about/accessibility/index.md:8]: Yeah, seems reputable enough to me. Let's do that.

File: src/_about/accessibility/index.md (hunk 4 above)
Comment (humancompanion-usds) [reply src/_about/accessibility/index.md:87]: You should be able to move it. Let me know if you have trouble.

File: src/_about/accessibility/index.md (hunk 5 above)
Comment (artsymartha68) [reply src/_about/accessibility/index.md:66]: Maximum. Wow. We've never had that luxury to even conceive of! I think I'd add
- TalkBack on an Android device (then the question comes which one from the millions available)
- NVDA with Chrome (we heard today from [the newest WebAIM survey](https://webaim.org/projects/screenreadersurvey10/) that NVDA is surpassing JAWS)

**Review by humancompanion-usds — APPROVED** [review]
